src/controllers/dashboard/dashboard.js

```diff
@@ -636,20 +636,23 @@ define(["datetime", "events", "itemHelper", "serverNotifications", "dom", "globa
         getNowPlayingImageUrl: function (item) {
             if (item && item.BackdropImageTags && item.BackdropImageTags.length) {
                 return ApiClient.getScaledImageUrl(item.Id, {
+                    maxWidth: Math.round(dom.getScreenWidth() * 0.20),
                     type: "Backdrop",
                     tag: item.BackdropImageTags[0]
                 });
             }
 
             if (item && item.ParentBackdropImageTags && item.ParentBackdropImageTags.length) {
                 return ApiClient.getScaledImageUrl(item.ParentBackdropItemId, {
+                    maxWidth: Math.round(dom.getScreenWidth() * 0.20),
```

Comment:
Almost feel like we should include this in the API client to avoid so much duplication.

Reply:
Not everything uses the screen width as a base though. Ideally, we'd get rid of all this for some sensible logic once moving to a framework.

Reply:
Why this `0.20` is still there? I thought you said you changed all the things...

Reply:
If it's currently impossible to compute, at least add a comment explaining what this `0.20` means and how it is generated, and a `// TODO` to fix it later.